js/src/util/validation.js

```diff
@@ -42,9 +44,14 @@ export function validateAbi (abi) {
   try {
     abiParsed = JSON.parse(abi);
 
-    if (!util.isArray(abiParsed)) {
+    if (!apiutil.isArray(abiParsed)) {
```

Comment:
While we're at it: I'd prefer `Array.isArray` here. It's less brittle and it's widely known so you don't have to jump to another place to find out what it does.